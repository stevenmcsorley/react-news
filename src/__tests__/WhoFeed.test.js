import React from 'react';
import { shallow } from 'enzyme';
import Who from '../components/news/WhoFeed'

test("testing Who feed",  async () =>{

    const WhoComp = shallow(<Who />, 'div');
    expect(WhoComp).toMatchSnapshot();

})