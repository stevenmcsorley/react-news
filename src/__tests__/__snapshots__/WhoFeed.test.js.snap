// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Who feed 1`] = `ShallowWrapper {}`;
